test/fail_compilation/fail3753.d

```diff
@@ -1,6 +1,7 @@
 /*
+TEST_OUTPUT:
 ---
-Error: cannot mix core.std.stdlib.alloca() and exception handling in _Dmain()
+fail_compilation/fail3753.d(35): Error: static assert:  `0` is false
```

Comment:
This test should use `DISABLED` instead of `version (linux) static assert(false)`

Reply:
#9234